src/cartridges/app_adyen_SFRA/cartridge/client/default/js/adyen/checkout/paymentMethodsConfiguration/googlePay/googlePayConfig.js

```diff
@@ -8,10 +8,13 @@ class GooglePayConfig {
     this.helpers = helpers;
   }
 
-  onSubmit = () => {
+  onSubmit = (state, component, actions) => {
     this.helpers.assignPaymentMethodValue();
     document.querySelector('button[value="submit-payment"]').disabled = false;
     document.querySelector('button[value="submit-payment"]').click();
+    actions.resolve({
+      resultCode: 'Pending',
```

Comment:
How does passing resultCode as 'Pending' fix the issue? Is it something documented on checkout webComponent? Could you also add some description in ticket on how this solves the issue?

Reply:
The component needs to be resolved using a result code, but since for google pay the payments call is not done in onSubmit, this needs to be hardcoded.